script/run-docker-container/_cm.yaml

```diff
@@ -16,6 +16,7 @@ category: Docker automation
 default_env:
   CM_DOCKER_DETACHED_MODE: 'yes'
   CM_DOCKER_REUSE_EXISTING_CONTAINER: 'no'
+  CM_DOCKER_PRIVILEGED_MODE: 'yes'
```

Comment:
Hi @anandhu-eng we should turn this off by default for general docker runs and turn it on inside run-mlperf-inference-app.